### ISSUE TITLE
Some Questions about reproducing the result on COCO by ResNet50-FPN

### ISSUE THREAD

**Issue by leohsuofnthu** [dbolya/yolact#341]
Hi,
Thanks for the amazing works!! Since I havnt seen any implementation in TF 2.0, I decide to try to reproduce YOLACT as my practice (https://github.com/leohsuofnthu/Tensorflow-YOLACT), I already finish the loss calculation part, but still feels confusing. It would be nice to have your help :)

1. So far, I am focusing on Resnet50-FPN to train from scratch on MSCOCO,  I am wondering what is the normal range of each loss during the training? 

2. For the part of Linear Combination Mask Loss,  we divide it by the number of batches or also number of positive matching anchors? 

3. I am quite new to Object detection, I wonder if I use Adam, does that affect mAP a lot?

Thanks for your help !

**Comment by leohsuofnthu** [comment]
Totally forgot that I can run the original code myself 😅
